lib/spack/spack/audit.py

```diff
@@ -298,7 +298,7 @@ def _check_build_test_callbacks(pkgs, error_cls):
 def _check_patch_urls(pkgs, error_cls):
     """Ensure that patches fetched from GitHub have stable sha256 hashes."""
     github_patch_url_re = (
-        r"^https?://github\.com/.+/.+/(?:commit|pull)/[a-fA-F0-9]*.(?:patch|diff)"
+        r"^https?://(?:patch-diff\.)?github(?:usercontent)?\.com/.+/.+/(?:commit|pull)/[a-fA-F0-9]*.(?:patch|diff)"
```

Comment:
Line too long 😢

Reply:
I thought flake8 automatically ignored URLs...

Reply:
Maybe you need a meta url regex? 😂

Reply:
Oh right, there's no way for flake8 to tell that this is supposed to be a URL lol